### PR TITLE
suppress antd change::: error

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

I couldn't stand the Antd error anymore. Definitely not optimal to override this, but our logging is currently harmed by the antd noise.

### Before

Screenshot 2021-04-13 at 11 10 34

### After 

Screenshot 2021-04-13 at 11 10 58

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

**Comment by yakkomajuri** [comment]
Example of this at work:

Screenshot 2021-04-13 at 11 14 54

**Comment by mariusandra** [comment]
This error is rather annoying.... but the solution feels way too hacky :).

Are we the only ones on the internet with this problem? Would upgrading antd help? Any issues you could reference where people complain about it before realising that patching `console.error` is the only way to go?

File: yarn.lock [hunk 1]
```diff
@@ -10047,21 +9769,10 @@ rc-tree@^4.0.0, rc-tree@~4.1.0:
...
-    classnames "^2.2.6"
-    rc-align "^4.0.0"
-    rc-motion "^2.0.0"
-    rc-util "^5.5.0"
-
-rc-trigger@^5.1.2, rc-trigger@^5.2.1:
-  version "5.2.1"
-  resolved "https://registry.yarnpkg.com/rc-trigger/-/rc-trigger-5.2.1.tgz#54686220b884ed1e0750c4f2411fbb34d4928c99"
-  integrity sha512-XZilSlSDnb0L/R3Ff2xo9C0Fho2aBDoAn8u3coM60XdLqTCo24nsOh1bfAMm0uIB1qVjh5eqeyFqnBPmXi8pJg==
+rc-trigger@^5.0.0, rc-trigger@^5.0.4, rc-trigger@^5.1.2, rc-trigger@^5.2.1, rc-trigger@^5.2.5:
```
Comment (yakkomajuri) [inline yarn.lock:9772]: this is the key line here, making sure all our deps use v5.2.5

**Comment by yakkomajuri** [comment]
Ok so thanks for pushing me here @mariusandra - this was indeed a quick and dirty solution. Provided how long the error had been there, as well as a faint memory of others struggling to fix it, I decided I wanted to get it over with quickly.

Ultimately the problem was `rc-trigger`, which Antd depends on. We had a whole lot of versions of it flying around, but this consolidates them into one. `yarn` wasn't successful at deduping this (which it should have been able to do), so I installed the latest version of `rc-trigger` and used https://www.npmjs.com/package/yarn-deduplicate (better than doing it manually).

So this should fix it, as well as I threw in a fix for a missing `key` in a `map`

**Review by Twixes — APPROVED** [review]
Nice! @yakkomajuri

**Review by mariusandra — APPROVED** [review]
This is indeed a much nicer fix! 🥳

**Comment by paolodamico** [comment]
This is amazing! Thanks @yakkomajuri & @mariusandra !

**Pull request merged**